### DISCUSSION TITLE
Add support to use different commands of the php-cli

### DISCUSSION THREAD

**Discussion by alexander-schranz** [php/frankenphp#1099] [Ideas]
I'm looking for a way to use `frankenphp php-cli` directly as a replacement for the local PHP.

Typical commands I use:

```bash
php -m # list extensions
php --version # list version
php -r 'echo 1;' # run inline code

# or use it to install composer (currently workaround it create install script)
curl --show-error --silent https://getcomposer.org/installer | ./frankenphp php-cli -- --1
```

The command currently expect only a file:

> Warning: Unknown: Failed to open stream: No such file or directory in Unknown on line

Looks at the current state the `frankenphp php-cli` only supports run a `.php` script. Would be nice to have also the other options available if possible. I understand that not all can be supported but maybe this most common commands are possible?

**Comment by dunglas** [comment]
Actually, we should be able to support most commands by copying more code from the CLI SAPI... It's on my todo list, but PR welcome is someone is quicker!

**Reply by alexander-schranz** [reply]
If we want to implement all. Maybe `php -S 0.0.0.0:8000 -t public config/router.php` could just point to start frankenphp / caddy implementation.

**Reply by dunglas** [reply]
Indeed, it can just forward to `frankenphp php-server`, but `-S` and other dev-only or weird options are not top priority.

**Reply by aszenz** [reply]
Frankenphp is great as a single static binary so I would also like to use it for both cli and web (locally and in GitHub actions) but currently it's not possible because of composer scripts.

I also couldn't get composer to use php from frankenphp in actions.
